pkg/provisioning/bootguard/bootguard.go

```diff
@@ -888,20 +888,41 @@ func (b *BootGuard) BPMKeyMatchKMHash() (bool, error) {
 	return true, nil
 }
 
-// SaneBPMSecurityProps verifies that BPM contains security properties set accordingly to spec
-func (b *BootGuard) SaneBPMSecurityProps() (bool, error) {
+func (b *BootGuard) StrictSaneBPMSecurityProps() (bool, error) {
 	switch b.Version {
 	case bgheader.Version10:
 		flags := b.VData.BGbpm.SE[0].Flags
-		if !flags.DMAProtection() {
-			return false, fmt.Errorf("dma protection should be enabled for bootguard")
-		}
```

Comment:
If this is strict, why remove the check here?

Reply:
This is not strict. It just how git/github renders this for you. Or wdy mean?

Reply:
The function name is `StrictSaneBPMSecurityProps`, but for me it seems the check `!flags.DMAProtection()` is removed from it and placed in the function `SaneBPMSecurityProps`. I would assume this check should be kept in the the function `StrictSaneBPMSecurityProps`, because the function name suggests that it is the more `strict` variant of `SaneBPMSecurityProps`.

Reply:
No. There was only one function SaneBPMSecurityProps where all checks where in, I added StrictSaneBPMSecurityProps and sorted the checks accordingly.

It's just GitHub showing it a bit weird for you.

Reply:
So the check just belongs to the `sane`-function and not to `strictsane`-function :)